packages/shared/src/components/search/SearchSourceItem.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { ReactElement } from 'react';
+import { ClickableText } from '../buttons/ClickableText';
+import { SearchChunkSource } from '../../graphql/search';
+
+interface SearchSourceItemProps {
+  item: SearchChunkSource;
+}
+
+export function SearchSourceItem({
+  item,
+}: SearchSourceItemProps): ReactElement {
+  if (!item) return null;
+
+  const { title, snippet, url } = item;
+
+  return (
+    <div className="w-60 laptop:w-full">
+      <ClickableText
```

Comment:
Can't remember, but was there not something wrong with this one?

Reply:
When wrapping it with other texts. Thankfully we don't have it here, though maybe we should now open a ticket to create the ideal link component.